simphony/cuds/abc_modeling_engine.py

```diff
@@ -22,6 +25,31 @@ class ABCModelingEngine(object):  # pragma: no cover
     """
     __metaclass__ = ABCMeta
 
+    def __init__(self, *args, **kwargs):
```

Comment:
> A user new to the framework will need to use get_wrapper(wrapper_name) in order to fetch the class and its docstring

This will be unnecessary. Users would need to fill a CUDS using the generated metadata classes and create and run a simulation with it.

> wrapper instance is created for each individual Simulation

Even though we currently create a new instance per simulation, we have no means of controlling whether multiple simulations have consistent configuration. To make it happen, we would need new parts in place. We can address some of these problems in client/server subsystem.

> What are these metadata that are required for instantiation?

The motivation behind instantiating wrappers internally, is to have CUDS object which contain all the necessary information about a computational model to start a simulation. Information such as engine name, parallelism, being internal or fileio, etc. This way, Simulation will have enough information to create an instance of a suitable wrapper (which is defined in the CUDS), configure it and then run it.

The other goal here, is to hide the complexity of the wrappers from users and move it to SimPhoNy side. This will help us to move things around internally without affecting the way SimPhoNy is being used.